backend/src/main/java/com/bether/bether/connection/discord/infrastructure/DiscordSlashCommandListener.java

```diff
@@ -15,7 +17,7 @@
 public class DiscordSlashCommandListener extends ListenerAdapter {
 
     private final WebConfigProperties webConfigProperties;
-    private final DiscordMessageSender discordMessageSender;
+    private final DiscordMessageBuilder discordMessageBuilder;
 
     public void onSlashCommandInteraction(final SlashCommandInteractionEvent event) {
```

Comment:
오버라이드 애너테이션을 추가하면 좋을 것 같습니다.

```suggestion
    @Override
    public void onSlashCommandInteraction(final SlashCommandInteractionEvent event) {
```

Reply:
좋습니다~~ 감사합니다